### PR TITLE
MDOT Order Accessories Page: State Setup and "Can't Reorder Batteries Until" Next Availability Date UI Screen

### PR DESCRIPTION
**Pull request by maharielrosario**
## Description
This PR is about refining the page where Veterans can order hearing aid accessories. 

## Screenshots
screenshot-localhost_3001-2020 04 27-22_41_35
screenshot-localhost_3001-2020 04 27-22_43_50

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/2346/components/SelectArrayItemsAccessoriesWidget.jsx [hunk 1]
```diff
@@ -7,80 +10,140 @@ import {
...
+    return (
+      <>
+        <h4>Select the hearing aid accessories you need</h4>
+        <p>
+          You'll be sent a 6-month supply for each accessory you choose below.
+        </p>
+        {supplies.map(
+          supply =>
+            supply.productGroup === HEARING_AID_ACCESSORIES &&
+            supply.availableForReorder === true ? (
```
Comment (bradl3yC) [inline src/applications/disability-benefits/2346/components/SelectArrayItemsAccessoriesWidget.jsx:47]: I think we should try to move away from all of the ternaries used here.  Before we do a map, we could do a filter and that way our array of products only includes the ones that we will show - from there we can map over those and output the JSX for those.  Something like:

```supplies.filter(supply => supply.productGroup === HEARING_AID_ACCESSORIES && supply.availableForReorder).map(supply => <jsx here>```

**Review by bradl3yC — CHANGES_REQUESTED** [review]
See comment

File: src/applications/disability-benefits/2346/components/SelectArrayItemsAccessoriesWidget.jsx (hunk 1 above)
Comment (mr0sari0) [reply src/applications/disability-benefits/2346/components/SelectArrayItemsAccessoriesWidget.jsx]: Thanks, just pushed an update for this.

**Review by bradl3yC — APPROVED** [review]

**Review by bradl3yC — APPROVED** [review]

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**